### PR TITLE
[docs] Train Controlnet SDXL - Add example command to train model on 24GB GPU

### PR DESCRIPTION
**Pull request by MDCurrent**
# What does this PR do?

Adding an example command to help outline how to train a controlnet on a 24GB GPU if the user does not have access to a 40GB A100

Happy to change verbiage/ close if not valuable but this is what worked for me running on a single 4090 going through this readme

https://wandb.ai/mdcurrent/sd_xl_train_controlnet/runs/tt9m4vhp/overview is the run using this command

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [ ] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

- Docs: @stevhliu and @yiyixuxu - if this is the wrong team to send it to happy to tag contributors to this file

### CODE REVIEW

File: examples/controlnet/README_sdxl.md [hunk 1]
```diff
@@ -89,6 +89,26 @@ To better track our training experiments, we're using the following flags in the
 
 Our experiments were conducted on a single 40GB A100 GPU.
 
+To train this model in a 24GB memory space we will need to lower the resolution to 512 and raise the gradient_accumulation_steps to 8
+
+```
+accelerate launch train_controlnet_sdxl.py\
```
Comment (patrickvonplaten) [inline examples/controlnet/README_sdxl.md:95]: wouldn't `gradient_checkpointing` allow going for full 1024 resolution on a 24GB GPU?

Comment (MDCurrent) [reply examples/controlnet/README_sdxl.md:95]: messed around last night and did not get a successful run https://wandb.ai/mdcurrent/sd_xl_train_controlnet/runs/5fvzz9qw/overview?workspace=user-mdcurrent. I’ll come back later today trying 786 or simmilar

Comment (MDCurrent) [reply examples/controlnet/README_sdxl.md]: was able to get this running with the command now committed but its definitely running close to oom
Screenshot 2023-10-19 at 2 10 07 PM

**Comment by patrickvonplaten** [comment]
Hey @MDCurrent,

I'm still quite positive that one could train the a SDXL + ControlNet using just 24GB using gradient checkpointing, 8bit Adam, ... so I'm not sure it's a good idea to convey the message that it's *impossible* to train on 1024 resolution with 24GB. Note that some people in the community seem to train SDXL on just 12 GB of memory: https://github.com/bghira/SimpleTuner/releases/tag/v0.7.0 so I'm pretty sure 24GB should be enough for ControlNet + SDXL at 1024 resolution

**Comment by MDCurrent** [comment]
@patrickvonplaten agreed, I’m still playing with this so i will close and will open another PR if i get something valuable to add. Thanks for the eyes/ comments

**Comment by pribadihcr** [comment]
Hi @MDCurrent, Any luck? thanks

**Pull request closed without merging**